doc/docfx.json

```diff
@@ -0,0 +1,77 @@
+{
+    "metadata": [
+        {
+            "src": [
+                {
+                    "files": [
+                        "source/scripting_v2/**.csproj"
+                    ],
+                    "exclude": [
+                        "**/bin/**",
+                        "**/obj/**",
+                        "_site/**"
+                      ],
+                      "src": "../"
+                }
+            ],
+            "dest": "scripting_v2"
+        },
+        {
+            "src": [
+                {
+                    "files": [
+                        "source/scripting_v3/**.csproj"
+                    ],
+                    "exclude": [
+                        "**/bin/**",
+                        "**/obj/**",
+                        "_site/**"
+                      ],
+                      "src": "../"
+                }
+            ],
+            "dest": "scripting_v3"
+        }
+    ],
+    "build": {
+        "content": [
+            {
+                "files": [
+                    "LICENSE.txt",
+                    "**.md",
+                    "**.yml",
+                    "scripting_v2/**.md",
+                    "scripting_v3/**.md",
+                    "scripting_v2/**.yml",
+                    "scripting_v3/**.yml",
+                    "scripting_v2/api/**.yml",
+                    "scripting_v3/api**.yml"
+                ]
+            }
+        ],
+        "dest": "_site",
+        "globalMetadataFiles": [],
+        "fileMetadataFiles": [],
```

Comment:
Are these two entries necessary, consider that they are empty?

Reply:
I've took them from the documentation example and they were there. I'll see what happens if I remove them